File: Menu.py
from Item import Item

class Menu:
    def __init__(self):
        self.__Item = []
    def return_menu(self):
        return self.__Item
    def return_price(self,s):
        for i in self.__Item:
            if(i.name == s):
                return i.price
        return None
    def add_Item_by_obj(self,Item):
        self.__Item.append(Item)
    def add_Item(self,s,value):
        item =  Item(s,value)
        self.__Item.append(item)

File: Menu.ts
import {Item} from "./Item"
import { Restaurant } from "./Restaurant";
export class Menu{
    private __Items : Array<Item> = [];
    constructor(){
    }
    getMenuItems(){
        return this.__Items;
    }
    getPrice(s:string){
        for (let i of this.__Items){
            if(i.name==s){
                return i.price;
            }
        }
    }
    addItem(Item : Item){
        this.__Items.push(Item);
    }
    addItembyName(name:string , price:number, type: number, restaurant : Restaurant){
        let item = new Item(name, price, type,restaurant);
        this.__Items.push(item);
    }
}